provider/virt_storage/backend/ceph.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from provider.virt_storage.helper import rbdcli
+
+from provider.virt_storage import storage_volume
+from provider.virt_storage.backend import base
+
+
+class CephPool(base.BaseStoragePool):
```

Comment:
suggest change ceph.py to rbd.py
CephPool -> RBDPool
to match https://libvirt.org/storage.html#StorageBackendRBD

Reply:
Have updated, thanks zhenchao.